test/integration/scheduler/preemption_test.go

```diff
@@ -129,6 +131,51 @@ var _ = ginkgo.Describe("Preemption", func() {
 			util.ExpectWorkloadsToBeAdmitted(ctx, k8sClient, cq.Name, highWl2)
 			util.ExpectWorkloadsToBePending(ctx, k8sClient, lowWl1, lowWl2)
 		})
+
+		ginkgo.It("Should preempt newer Workloads with the same priority when there is not enough quota", func() {
+			ginkgo.By("Creating initial Workloads with different priorities")
+			wl1 := testing.MakeWorkload("wl-1", ns.Name).
+				Queue(q.Name).
+				Priority(lowPriority).
+				Request(corev1.ResourceCPU, "1").
```

Comment:
```suggestion
				Request(corev1.ResourceCPU, "2").
```

Then we don't need wl2

Reply:
Having less workloads could make the test marginally faster, but it's not a big deal.
I'm ok either way.